intranet/templates/eighth/emails/activity_cancelled.html

```diff
@@ -0,0 +1,14 @@
+<html><body>
+{% with description="You need to update your Eighth Period signup selections." target=info_link name="View status" %}
+    {% include "email_metadata.html" %}
+{% endwith %}
+
+<p>
+    The activity you are signed up for on {{ date_str }} during {{ sched_act.block.block_letter }} block ({{ sched_act.activity.name }}) has been cancelled. You will need to select a new activity.
```

Comment:
Because I put the name in parentheses, it will look weird to use `full_title`  (which adds prefixes like `Special:`).

Also, `full_title` will add a `(Cancelled)` suffix, which is redundant when we're emailing them specifically to say that the activity is cancelled.

Reply:
Ok, makes sense.